internal/stream/stream.go

```diff
@@ -1,4 +1,4 @@
-package worker
+package stream
```

Comment:
we have two stream impls. One in `web/` and one in `worker/`. Should this be sub-dir'd accordingly? In future, we can move the web part as well here.

Reply:
Yes, Didn't touch the web part yet but we can move that here slowly